app/views/release_notes/_v4.7.1.html.erb

```diff
@@ -0,0 +1,4 @@
+ <h3>AACT 4.7.1 <i>(September 18, 2021)</i></h3>
```

Comment:
This release has incorrect notes. They should be:

- Renamed ctgov_beta_group_code to ctgov_group_code
- Added mesh_type column for browse_conditions and browse_interventions tables
- Edited code to save more MeSH types to the database
- changed factory_girl to the factory_bot gem
- added Airbrake for tracking errors
- Added database explanation and visualization to the README
- Edited how the downcase mesh terms are created so they would be updated regularly
- Edited the way the loads are run to better switch between schemas
- Added environmental variables for the ctgov_beta schema and database

Remove the current release notes from this view and add the ones I listed above instead